### DISCUSSION TITLE
Improve relational interface now that the new table interface got merged

### DISCUSSION THREAD

**Discussion by LasseRosenow** [directus/directus#15225] [Feature Requests]
Recently the new table interface was merged (#12820)
But it created some inconsistencies and visual design issues.

### Problem
There seems to be a huge gap between the `field name` and the `entries` now. This does not look too bad in the table layout, as it has a border, but in the normal `list layout` it looks very awkward.
### Solution 1 (Hacky)
The `.one-to-many` class could get a `margin-top: -3rem;`.
### Solution 2 (Good)
What if we change the interface API so that it is possible to for example inject html in up to 75% of the space next to the field name?

Then the layout could be exactly the same as it is for collections and with position absolute we could manually make the search overflow the field name in its extended state, if the space is required.

I would make a PR trying if this approach works if you think that would be acceptable 🙂
<table>
<tr>
<th>How it is</th>
<th>How it could be</th>
</tr>
<tr>
<td width="50%">

![grafik](https://user-images.githubusercontent.com/10547444/186214713-b42a2425-59f6-48e8-b6ae-5c63836ee84c.png)

</td>
<td>

![grafik](https://user-images.githubusercontent.com/10547444/186215312-8e2352c3-a6bc-4588-9481-6f6ca9c1572a.png)

</td>
</tr>
</table>

## 2. The round buttons are not everywhere.
Now since the new layout we not only have the round action buttons for the table, but also for the list layout, but for the m2a layout its still the old buttons, which seems quite inconsistent.

![grafik](https://user-images.githubusercontent.com/10547444/186214587-0ffe849f-24c7-4d3c-bfaa-163ed7a46898.png)

**Comment by rijkvanzanten** [comment]
I agree. I'm also not sold on using the round buttons in the first place[^1], so I think we might revert the list style to the previous "bottom button" style 👍🏻 

[^1]: Those used to be exclusively used for header buttons, and I think we should keep it that way

**Reply by rijkvanzanten** [reply]
cc @azrikahar

**Reply by LasseRosenow** [reply]
Actually I do like the new design with the round buttons at the top right, at it is easier to reach with the mouse. Like you don't have to scroll to the end of the list first. But yeah looks a bit odd as long as we have this weird gap ;)

**Reply by azrikahar** [reply]
@LasseRosenow you can also check out the prior discussion that was had regarding it: https://github.com/directus/directus/pull/12820#issuecomment-1116075791. A major one was more so accommodating the search bar (especially when the interface is half width so the search bar will lean to the left even more), and another minor one would be removing the gap at it's current state would mean it'll take up space over the form label div, in terms of elements. 

That being said, this definitely something that can be discussed further with future iterations!

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
